src/lib/schemeUtils.ts

```diff
@@ -98,7 +105,7 @@ export function schemeName(scheme: ISchemeState|IContractInfo, fallback?: string
       }
       const genericSchemeInfo = genericSchemeRegistry.getSchemeInfo(contractToCall);
       if (genericSchemeInfo) {
-        name = genericSchemeInfo.specs.name;
+        name = splitCamelCase(genericSchemeInfo.specs.name);
```

Comment:
1. calling splitCamelCase on all genericScheme with no alias is not good.  for example : no need to call splitCamelCase on `DutchX` .
2. why using splitCamelCase at all here  ?

Reply:
Yeah, sorry, thought I reverted all of it.  Created a PR to fix, here: https://github.com/daostack/alchemy/pull/1822